model-optimizer/extensions/middle/EltwiseInputReshape.py

```diff
@@ -88,35 +88,20 @@ def find_and_replace_pattern(self, graph: Graph):
                     else:
                         mapping[new_shape].append(consumer)
 
-            if node.has_valid('value'):
-                # Check that requested shape are the same
-                # In case if they are different, we duplicate them
-                for shape_key in mapping.keys():
-                    shape = list(shape_key)
-                    new_value = np.reshape(node.value, shape)
-                    node_copy = Op.create_input_data_node(graph, node.id + '/copy', value=np.array(new_value))
-                    for consumer in mapping[shape_key]:
-                        edge_attrs = graph.get_edge_data(node.id, consumer.id)[0]
-                        del edge_attrs['new_shape']
-
-                        # Remove edge from previous data node and connect new data node with its consumer
-                        graph.remove_edge(node.id, consumer.id)
-                        graph.add_edge(node_copy.id, consumer.id, **edge_attrs)
-            else:
-                # Insert Reshape layer between data node and consumer
-                for shape_key in mapping.keys():
-                    shape = list(shape_key)
-                    reshape_name = node.soft_get('name', node.id) + '/EltwiseReshape'
-                    reshape = Reshape(graph, attrs={'name': reshape_name})
-                    reshape_dim = Const(graph,
-                                        {'value': shape, 'name': reshape_name + '/Shape'}).create_node_with_data()
-                    reshape_data = reshape.create_node_with_data(inputs=[node, reshape_dim])
-
-                    # Iterate over consumers and reconnect them to Reshape layer output
-                    for consumer in mapping[shape_key]:
-                        edge_attrs = graph.get_edge_data(node.id, consumer.id)[0]
-                        del edge_attrs['new_shape']
-
-                        # Reconnect edge from original data node to Reshape output datanode
-                        graph.remove_edge(node.id, consumer.id)
-                        graph.add_edge(reshape_data.id, consumer.id, **edge_attrs)
+            # Insert Reshape layer between data node and consumer
+            for shape_key in mapping.keys():
+                shape = list(shape_key)
+                reshape_name = node.soft_get('name', node.id) + '/EltwiseReshape'
+                reshape = Reshape(graph, attrs={'name': reshape_name})
```

Comment:
use `create_op_with_const_inputs` function

Reply:
this is not suitable. It needs simultaneous creation of data node for `Reshape`.